packages/gatsby-theme-code-examples/src/multi-code-example.js

```diff
@@ -7,18 +7,18 @@ import {
 } from '@commercetools-docs/ui-kit';
 import useCodeExamples from './use-code-examples';
 
+const isCodeExampleNode = (alienNode) =>
```

Comment:
In a couple of place in the code, we need to check what's the React type of a specific component. Unfortunately `component.type`, in case of functional components is, indeed, a function which name gets mangled in minification. 
`component.type.name === 'MyComponent'` works in dev mode but not once built.
So the problem I'm trying to address here is to `guess` if the component is a `CodeExample` component by checking ig the mandatory `href` prop is defined.

It's weak but I couldn't find any better soultion

Reply:
Good enough!.  As said, it's just a small optimization to catch most common types of errors before the dev server crashes or the app breaks otherwise